serializations.md

```diff
@@ -630,6 +631,7 @@ The payload is a serialized signed channel off-chain transaction and can not be
 
 
 #### Channel solo force progress transaction
+version 1:
```

Comment:
If I read this section, it looks both versions 1 and 2 are legal in Roma and Minerva and Fortuna.

Reply:
I see "version 2 (valid from Fortuna hardfork on)". Ok.